sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmosdb/CosmosDbReactiveRepositoriesAutoConfigureRegistrar.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.microsoft.azure.spring.autoconfigure.cosmosdb;
+package com.azure.spring.autoconfigure.cosmosdb;
 
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.EnableReactiveCosmosRepositories;
 import com.microsoft.azure.spring.data.cosmosdb.repository.config.ReactiveCosmosRepositoryConfigurationExtension;
```

Comment:
why these cosmos releated packages didn't update to `azure.azure`?

Reply:
Moary will update this in his pr. Maybe need to discuss this with xiaolu.